ONSdigital/florence
# Improve client-side logging

### What

Improved some of the logging in Florence to help us when a content loss issue happens:
- Add logging on 'GET content' to help get coverage on what information is coming back from Zebedee after a 'POST content'.
- Log timestamps on post content, get content, disable Florence UI and enable Florence UI.
- Remove ping request results from the console and store in a global variable.

### How to review

1) Switch to the branch `feature/logging`
2) Open a bulletin in the editor and check the GET and POST request for the page content are logged.
3) Pings should no longer show in the console but are available with the `Florence.ping` global object.

### Who can review

Anyone but me.
